Migrations/MyPlaceDbContext.cs

```diff
@@ -0,0 +1,27 @@
+using rexfinder_api.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace rexfinder_api.Migrations;
+
+public class MyPlaceDbContext : DbContext
```

Comment:
Rename MyPlacesDbContext (make plural)

Reply:
Ok, I will make the change and resubmit.